core/embed/sys/mpu/stm32u5/mpu.c

```diff
@@ -161,35 +155,8 @@ extern uint32_t _codelen;
 #define COREAPP_FLASH_SIZE \
   (FIRMWARE_MAXSIZE - (COREAPP_FLASH_START - KERNEL_FLASH_START))
 
-#define KERNEL_RAM_START (SRAM2_BASE)
-#define KERNEL_RAM_SIZE (KERNEL_SRAM2_SIZE)
-
-#ifdef STM32U585xx
-#define COREAPP_RAM1_START SRAM1_BASE
-#define COREAPP_RAM1_SIZE (SRAM1_SIZE - 512)
-
-#define COREAPP_RAM2_START (SRAM2_BASE + KERNEL_SRAM2_SIZE + KERNEL_U_RAM_SIZE)
-#define COREAPP_RAM2_SIZE                                            \
-  (SRAM2_SIZE - KERNEL_SRAM2_SIZE - KERNEL_U_RAM_SIZE + SRAM3_SIZE - \
-   FRAMEBUFFER_SRAM_SIZE)
-#else
-#define COREAPP_RAM1_START SRAM5_BASE
-#define COREAPP_RAM1_SIZE SRAM5_SIZE
-#endif
-
-#else
-
-#ifdef STM32U585xx
-#define MAIN_SRAM_START SRAM2_BASE
-#define MAIN_SRAM_SIZE SRAM2_SIZE
-#define AUX_SRAM_START SRAM1_BASE
-#define AUX_SRAM_SIZE (SRAM1_SIZE - BOOTARGS_SIZE)
-#else
-#define MAIN_SRAM_START SRAM2_BASE
-#define MAIN_SRAM_SIZE SRAM2_SIZE
-#define AUX_SRAM_START SRAM5_BASE
-#define AUX_SRAM_SIZE SRAM5_SIZE
-#endif
+#define COREAPP_RAM1_START AUX_RAM_START
```

Comment:
`COREAPP_RAM1_START` and `COREAPP_RAM1_SIZE` are no longer used.

Reply:
removed https://github.com/trezor/trezor-firmware/commit/5e57d8d4626a6ad0dbd44ae630dd040781c64f13